### ISSUE TITLE
antew/vim-elm-language-server merged into w0rp/ale

### ISSUE THREAD

**Issue by andys8** [leojpod/nvim-config#1]
Once https://github.com/w0rp/ale/issues/2592 is merged and the plugin updated, `antew/vim-elm-language-server` can be replaced. The linter name is `elm_ls`. See https://github.com/elm-tooling/elm-language-server/pull/80/files for changes.

If you run the `elm-language-server` with `coc` you probably don't need both of those plugins.

```
Plug 'antew/vim-elm-analyse', { 'do': 'npm i -g elm-analyse' }
Plug 'antew/vim-elm-language-server'
```

**Comment by leojpod** [comment]
Quite true cheers! 

My nvim setup is a bit messy at the moment, I've started to move to coc but didn't complete the process fully. I'll make your adjustment right away thanks!
